doc/source/api/index.rst

```diff
@@ -1,5 +1 @@
-Overview
```

Comment:
Please revert this, we eventually do want to give some type of overview description of the documentation.